do "hgshow 539358" to see where they got moved!

#diff -r ac8a73f9dbb7 toolkit/content/jar.mn
#--- a/toolkit/content/jar.mn	Sat Jun 09 00:53:46 2018 +0300
#+++ b/toolkit/content/jar.mn	Sat Jun 09 05:10:28 2018 +0200
#@@ -106,5 +106,3 @@
# #ifdef XP_MACOSX
#    content/global/macWindowMenu.js
# #endif
#-   content/global/gmp-sources/openh264.json    (gmp-sources/openh264.json)
#-   content/global/gmp-sources/widevinecdm.json (gmp-sources/widevinecdm.json)
##diff -r 2e3e89c9c68c toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm
##--- a/toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm	Wed Sep 26 00:49:06 2018 +0300
##+++ b/toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm	Wed Sep 26 10:34:36 2018 +0200
##@@ -6,13 +6,15 @@
## 
## /* exported ProductAddonChecker */
## 
##-const LOCAL_GMP_SOURCES = [{
##-  "id": "gmp-gmpopenh264",
##-  "src": "chrome://global/content/gmp-sources/openh264.json",
##-}, {
##-  "id": "gmp-widevinecdm",
##-  "src": "chrome://global/content/gmp-sources/widevinecdm.json",
##-}];
##+const LOCAL_GMP_SOURCES = [
##+//{
##+//  "id": "gmp-gmpopenh264",
##+//  "src": "chrome://global/content/gmp-sources/openh264.json",
##+//}, {
##+//  "id": "gmp-widevinecdm",
##+//  "src": "chrome://global/content/gmp-sources/widevinecdm.json",
##+//}
##+];
## 
## var EXPORTED_SYMBOLS = [ "ProductAddonChecker" ];
## 
#diff -r e704e3565db9 toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm
#--- a/toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm	Thu Jul 11 00:49:58 2019 +0300
#+++ b/toolkit/mozapps/extensions/internal/ProductAddonChecker.jsm	Thu Jul 11 03:32:36 2019 +0200
#@@ -7,14 +7,14 @@
# /* exported ProductAddonChecker */
# 
# const LOCAL_GMP_SOURCES = [
#-  {
#-    id: "gmp-gmpopenh264",
#-    src: "chrome://global/content/gmp-sources/openh264.json",
#-  },
#-  {
#-    id: "gmp-widevinecdm",
#-    src: "chrome://global/content/gmp-sources/widevinecdm.json",
#-  },
#+//  {
#+//    id: "gmp-gmpopenh264",
#+//    src: "chrome://global/content/gmp-sources/openh264.json",
#+//  },
#+//  {
#+//    id: "gmp-widevinecdm",
#+//    src: "chrome://global/content/gmp-sources/widevinecdm.json",
#+//  },
# ];
# 
# var EXPORTED_SYMBOLS = ["ProductAddonChecker"];
diff -r 08cd64cdbc3b toolkit/content/jar.mn
--- a/toolkit/content/jar.mn	Fri Jul 17 11:18:09 2020 +0000
+++ b/toolkit/content/jar.mn	Sun Jul 19 01:56:47 2020 +0200
@@ -103,8 +103,6 @@
 #ifdef XP_MACOSX
    content/global/macWindowMenu.js
 #endif
-   content/global/gmp-sources/openh264.json    (gmp-sources/openh264.json)
-   content/global/gmp-sources/widevinecdm.json (gmp-sources/widevinecdm.json)
 
 # Third party files
    content/global/third_party/d3/d3.js         (/third_party/js/d3/d3.js)
diff -r 08cd64cdbc3b toolkit/modules/GMPInstallManager.jsm
--- a/toolkit/modules/GMPInstallManager.jsm	Fri Jul 17 11:18:09 2020 +0000
+++ b/toolkit/modules/GMPInstallManager.jsm	Sun Jul 19 02:01:23 2020 +0200
@@ -53,14 +53,14 @@
 }
 
 const LOCAL_GMP_SOURCES = [
-  {
-    id: "gmp-gmpopenh264",
-    src: "chrome://global/content/gmp-sources/openh264.json",
-  },
-  {
-    id: "gmp-widevinecdm",
-    src: "chrome://global/content/gmp-sources/widevinecdm.json",
-  },
+//  {
+//    id: "gmp-gmpopenh264",
+//    src: "chrome://global/content/gmp-sources/openh264.json",
+//  },
+//  {
+//    id: "gmp-widevinecdm",
+//    src: "chrome://global/content/gmp-sources/widevinecdm.json",
+//  },
 ];
 
 function downloadJSON(uri) {
